doc/faq.md

```diff
@@ -28,6 +28,12 @@ That said, it is definitely possible to have write patterns that defeat this. De
 
 Theoretically, definitely. In Git, for example, the concept of "shallow clones" exists, and we could do something similar in Noms. This has not been implemented yet.
 
+### What about conflict resolution?
```

Comment:
Sorry, I'm going to rewrite this a little:

### How does Noms handle conflicts?

Currently, Noms doesn't handle them at all. But in the near future, we plan to add [automatic merge policies](link) for several common cases of conflicts. 

Similar to how Git uses simple heuristics to merge many classes of edits to same same file/directory, we believe that we can use some simple heuristics to merge Noms data. In fact, we think we can even do better than Git, because in general Noms data is more granular than data in Git. We have structs, sets, lists, and maps, where Git just has directories and files.

Whether to use an auto-merge policy will be something that developers choose to use on a case-by-case basis, and can always opt out of for more complex scenarios.

### Why don't you use CRDTs?

<your answer>